modules/rgbd/include/opencv2/rgbd/intrinsics.hpp

```diff
@@ -0,0 +1,76 @@
+#ifndef __OPENCV_RGBD_INTRINSICS_HPP__
+#define __OPENCV_RGBD_INTRINSICS_HPP__
+
+#include "opencv2/core/matx.hpp"
+
+namespace cv
+{
+    namespace kinfu
+    {
+struct Intr
```

Comment:
> Intr

Use meaningful names instead.

Reply:
This was existing code that was moved to a different file and will cause a change in variable name in a lot of files.